build.rs

```diff
@@ -1,6 +1,6 @@
-use tonic_build;
+use anyhow::{Context, Result};
+use tonic_build::compile_protos;
```

Comment:
This is probably fine but maybe keep fully qualified in bigger files.

Reply:
Sure, agree. :)